### PR TITLE
r.object.geometry: add json support

### PR DESCRIPTION
**Pull request by kritibirda26**
Use parson to add json support to r.object.geometry. The JSON output looks like as follows:

```json
[
    {
        "category": 27518,
        "area": 54,
        "perimeter": 40,
        "compact_circle": 1.5355295532059356,
        "compact_square": 0.73484692283495345,
        "fd": 1.8495247341090328,
        "mean_x": 631611.11111111112,
        "mean_y": 220648.14814814815
    },
    {
        "category": 27529,
        "area": 57,
        "perimeter": 36,
        "compact_circle": 1.3451172460704992,
        "compact_square": 0.83887049280786108,
        "fd": 1.772672742164326,
        "mean_x": 643460.52631578944,
        "mean_y": 217232.45614035087
    },
    {
        "category": 27605,
        "area": 9,
        "perimeter": 16,
        "compact_circle": 1.5045055561273502,
        "compact_square": 0.75,
        "fd": 2.5235914062700342,
        "mean_x": 640916.66666666663,
        "mean_y": 226527.77777777778
    },
    {
        "category": 27608,
        "area": 17,
        "perimeter": 24,
        "compact_circle": 1.6420328794169217,
        "compact_square": 0.68718427093627676,
        "fd": 2.2433805468090191,
        "mean_x": 641367.6470588235,
        "mean_y": 227926.4705882353
    }
]
```

### CODE REVIEW

**Comment by kritibirda26** [comment]
@cwhite911 this PR is ready for review.

**Review by cwhite911 — APPROVED** [review]
@kritibirda26 this looks good to me!

**Comment by kritibirda26** [comment]
@echoix can you please merge this PR?

File: raster/r.object.geometry/testsuite/test_r_object_geometry.py
Comment (echoix) [inline raster/r.object.geometry/testsuite/test_r_object_geometry.py:1]: Why was that file renamed already?

Comment (echoix) [reply raster/r.object.geometry/testsuite/test_r_object_geometry.py:1]: Never mind, it doesn't matter for collection, only the folder name.

**Review by echoix — APPROVED** [review]
